jenkins/spark-tests.sh

```diff
@@ -161,10 +161,26 @@ export CUDF_UDF_TEST_ARGS="--conf spark.rapids.memory.gpu.allocFraction=0.1 \
 --conf spark.pyspark.python=/opt/conda/bin/python \
 --py-files ${RAPIDS_PLUGIN_JAR}"
 
+
 export SCRIPT_PATH="$(pwd -P)"
 export TARGET_DIR="$SCRIPT_PATH/target"
 mkdir -p $TARGET_DIR
 
+run_delta_lake_tests() {
+  echo "run_delta_lake_tests SPARK_VER = $SPARK_VER"
+  SPARK_321_PATTERN="(3\.2\.[1-9])"
```

Comment:
minor side note, CDH is 3.2.1 based but I don't know if it has delta lake with it, assume separate issue anyway but just mentioning